tests/data/test-data.sql

```diff
@@ -238,7 +242,11 @@ insert into game_player_stats (gameId, playerId, AI, faction, color, team, place
   (41938, 1, 0, 0, 0, 2, 0, 1500, 500, NOW() + interval 3 minute),
   (41939, 1, 0, 0, 0, 2, 0, 1500, 500, NOW() + interval 4 minute),
   (41940, 1, 0, 0, 0, 2, 0, 1500, 500, NOW() + interval 5 minute),
-  (41941, 1, 0, 0, 0, 2, 0, 1500, 500, NOW() + interval 6 minute);
+  (41941, 1, 0, 0, 0, 2, 0, 1500, 500, NOW() + interval 6 minute),
+  (41945, 2, 0, 0, 0, 1, 0, 1500, 500, NOW() + interval 1 minute),
```

Comment:
The `scoreTime` you enter is *before* the `startTime` of the game.
Probably won't ever be a problem but maybe it's nice to keep them in sync as before (i.e. increment them all by six).

Reply:
Updated those lines for consistency